src/frontend/src/components/common/Alert/_alert.scss

```diff
@@ -18,6 +18,12 @@
     border-color: #ffeeba;
   }
 
+  &.alert-banner-warning {
+    color: #856404;
```

Comment:
shouldn't all these color values come from the defined variables?

Reply:
yes, let me change it